src/components/create-study-dialog/import-parameters.tsx

```diff
@@ -0,0 +1,58 @@
+import { FlatParameters } from '@gridsuite/commons-ui';
+import { makeStyles } from '@mui/styles';
+import { GridStudyTheme } from 'components/app-wrapper.type';
+import { FunctionComponent, useCallback, useState } from 'react';
+import { AdvancedParameterButton } from './advanced-parameter-button';
+import { CaseImportParameters } from 'services/network-conversion';
+import { Divider } from '@mui/material';
+
+export interface ImportParametersProps {
+    formatWithParameters: CaseImportParameters[];
+    currentParameters: Record<string, any>;
+    onChange: (paramName: string, value: any, isEdit: boolean) => void;
+}
+
+const useStyles = makeStyles<GridStudyTheme>((theme) => ({
+    paramDivider: {
+        marginTop: theme.spacing(2),
+    },
+}));
+
+export const ImportParameters: FunctionComponent<ImportParametersProps> = (
+    props
+) => {
+    const { formatWithParameters, onChange, currentParameters } = props;
+    const classes = useStyles();
+
+    const [areParamsDisplayed, setAreParamsDisplayed] = useState(false);
+
+    const handleShowParametersForCaseFileClick = useCallback(() => {
+        setAreParamsDisplayed((oldValue) => !oldValue);
+    }, []);
+
+    return (
+        <>
+            <Divider className={classes.paramDivider} />
+            <div
```

Comment:
Remove this and put a sx instead the Divider no ?

Reply:
I think it's a bit different, I would agree if it was a span, but here it would change the HTML structure

I'm replacing it with Box + sx though

Reply:
Ok